Rakefile

```diff
@@ -8,3 +8,52 @@ RSpec::Core::RakeTask.new(:test) do |t|
   t.rspec_opts = ['--color']
   t.pattern = 'spec/'
 end
+
+task :generate_host_config do |t, args|
+  if ENV["BEAKER_CONFIG"]
+    next
+  end
+  if not ENV['TEST_TARGET']
+    fail "FATAL: must set TEST_TARGET environment variable."
+  end
+
```

Comment:
we don't want a default for this?  
orch:
`ENV['TEST_TARGET'] || 'debian7-64af-64af-64af-64af'`